webapp/channels/src/packages/mattermost-redux/src/action_types/general.ts

```diff
@@ -13,6 +13,9 @@ export default keyMirror({
     CLIENT_LICENSE_RESET: null,
 
     CUSTOM_PROFILE_ATTRIBUTES_RECEIVED: null,
+    CUSTOM_PROFILE_ATTRIBUTES_CREATED: null,
+    CUSTOM_PROFILE_ATTRIBUTES_DELETED: null,
+    CUSTOM_PROFILE_ATTRIBUTES_PATCHED: null,
```

Comment:
nit: I'd change the names here to refer to the custom profile attribute fields specifically